plugin/storage/es/options.go

```diff
@@ -41,6 +41,7 @@ const (
 	suffixTagsAsFieldsAll   = suffixTagsAsFields + ".all"
 	suffixTagsFile          = suffixTagsAsFields + ".config-file"
 	suffixTagDeDotChar      = suffixTagsAsFields + ".dot-replacement"
+	suffixWildcardSearch    = ".wildcard-search"
```

Comment:
this name seems overly broad. Are you really supporting wildcard search for everything, or just for the service name?